### DISCUSSION TITLE
Live search in filemanager and show file/image link in filemanager

### DISCUSSION THREAD

**Discussion by zar00731** [UniSharp/laravel-filemanager#1176] [Ideas]
I am a Web Developer,
I have used this package in some of my projects. I feel very in need of live search and link show of uploaded files/images in filemanager.
This search will help us to search previously uploaded images for any new work. We can search by name and use one image in multiple places like WordPress Media Library.
If the link of the files or images will be shown there then we can copy it and use file links where we need actually.

I request to Team UniSharp to add these two features by thinking the importance of them.

Thanks
Jayed

**Comment by streamtw** [comment]
Search functionality is added in `v2.12.0` .
